app/models/cda/application_hearing.rb

```diff
@@ -12,5 +12,9 @@ def hearing_days
     def defence_counsels_on(date)
       defence_counsels.select { it.attended_on?(date) }
     end
+
+    def location
```

Comment:
I am not entirely convinced that naming this method `location` is the best choice, it's in the model and represents our domain/business logic, not presentation concerns.
Also, it's so generic,  if we search for "location" we'll get hundreds of occurrences.
An alternative might be `court_centre_name`, `court_centre_location`, ...